eam-light-backendweb/src/main/java/ch/cern/cmms/eamlightweb/rest/Monitoring.java

```diff
@@ -26,22 +28,21 @@
     @Path("/")
     @Produces("application/json")
     @Consumes("application/json")
-    public Response monitor(@QueryParam("equipment") String equipment, @QueryParam("workorder") String number) {
+    public Response monitor(@QueryParam("equipment") String equipment, @QueryParam("workorder") String number,
+        @QueryParam("equipmentUpdate") String equipmentUpdate, @QueryParam("workorderUpdate") String workorderUpdate) {
         Map<String, String> responses = new HashMap<>();
-        boolean check = true;
         try {
-            responses = monitoringService.monitoring(equipment, number, authenticationTools.getInforContext());
-        } catch (InforException e) {
```

Comment:
InforException being handled here is actually correct. Other types of exceptions should be seen as a problem with the monitoring service